pkg/blackboxexporter/blackboxexporter.go

```diff
@@ -12,6 +11,7 @@ import (
 	"k8s.io/apimachinery/pkg/util/intstr"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
+	blackboxConst "github.com/openshift/route-monitor-operator/pkg/consts/blackboxexporter"
```

Comment:
Why did you make the package uppercase?
```suggestion
	blackboxconst "github.com/openshift/route-monitor-operator/pkg/consts/blackboxexporter"
```